srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -23,6 +25,12 @@
     private static final Logger fLogger = LoggerFactory.getLogger(RpcVistaPatientDao.class);
     private static final String NO_WEIGHT = "0^NO WEIGHT ENTERED WITHIN THIS PERIOD";
     private static final String SPLIT_REGEX = "[\\s]+";
+    
+    // Codes to pass back to the controller for the JSON result
```

Comment:
Remember hexagonal architecture. This code should have no knowledge of web controllers or JSON.